### ISSUE TITLE
VxMark: order of candidates should be consistent in ballot order on review screen and printed ballot

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite#4919]

**Comment by adghayes** [comment]
Closing this in favor of clearer #6607.
